code/script.js

```diff
@@ -1,19 +1,164 @@
-// Start here
-
 // Step 1 - Welcome and introduction
-// Your code goes here
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 )
+const firstName = prompt("Please enter your name:");
+alert (
+  `Hello, ${firstName}! Let's get started.`);    
 
 // Step 2 - Food choice
-// Your code goes here
+alert(`Time to choose what food you want!`)
+const foodChoice = prompt(`What type of food would you like to order? 
+Enter a number: 
+1 - Pizza 
+2 - Pasta 
+3 - Salad
+`);
+
+let selectedFoodChoice = "";
+if (foodChoice === "1") {
+  selectedFoodChoice = "Pizza";
+  alert(`You've chosen ${selectedFoodChoice}!`);
+} else if (foodChoice === "2") {
+  selectedFoodChoice = "Pasta";
+  alert(`You've chosen ${selectedFoodChoice}!`);
+} else if (foodChoice === "3") {
+  selectedFoodChoice = "Salad";
+  alert(`You've chosen ${selectedFoodChoice}!`);
+} else {
+  alert("Invaild choice. Please select a number between 1 and 3.");
+}
+
+
+// Step 3 - Subtype food choice
```

Comment:
Do you think it would be easier if the "step 3" would include more comments in order to help next person? I was thinking about this with my project and don't know what's the "right" thing to do?

Reply:
True! Good point.